packages/top-app-bar-fixed/README.md

```diff
@@ -0,0 +1,136 @@
+> browsers including Chrome, Firefox, Safari, Edge, and IE11, but an additional
+> tooling step is required to resolve *bare module specifiers*, as well as
+> transpilation and polyfills for Edge and IE11. See
+> [here](https://github.com/material-components/material-components-web-components#quick-start)
+> for detailed instructions.
+
+## Example Usage
+
+### Standard
+
+<img src="images/standard.png" height="542px">
+
+```html
+<mwc-top-app-bar-fixed>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Dense
+
+<img src="images/dense.png" height="542px">
+
+```html
+<mwc-top-app-bar-fixed dense>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Prominent
+
+<img src="images/prominent.png" height="542px">
+
+```html
+<mwc-top-app-bar-fixed prominent>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Prominent and Dense
+
+<img src="images/prominent_and_dense.png" height="542px">
+
+```html
+<mwc-top-app-bar-fixed prominent dense>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Customize Colors
+
+<img src="images/custom_colors.png" height="542px">
+
+```css
+mwc-top-app-bar-fixed {
+  --mdc-theme-primary: orange;
+  --mdc-theme-on-primary: black;
+}
+```
+
+## API
+
+### Slots
+| Name | Description
+| ---- | -----------
+| `actionItems` | A number of `<mwc-icon-button>` elements to use for action icons on the right side.
+| `navigationIcon` | `<mwc-icon-button>` elements to use for the left icon.
+| `title` | Element that will be used as the title text.
```

Comment:
Does this work with anything other than div? like will span style correctly, etc.

Reply:
A span work work as well, as the slot is inside of a `flex`d span.

I'm wondering if this even needs to be a slot, or if it should instead just be an attribute.